ferry.go

```diff
@@ -508,9 +534,13 @@ func (f *Ferry) Start() error {
 	// If we don't set this now, there is a race condition where Ghostferry
 	// is terminated with some rows copied but no binlog events are written.
 	// This guarentees that we are able to restart from a valid location.
-	f.StateTracker.UpdateLastResumableBinlogPosition(pos)
+	f.StateTracker.UpdateLastResumableSourceBinlogPosition(sourcePos)
 	if f.inlineVerifier != nil {
-		f.StateTracker.UpdateLastResumableBinlogPositionForInlineVerifier(pos)
+		f.StateTracker.UpdateLastResumableSourceBinlogPositionForInlineVerifier(sourcePos)
+
+		if !f.Config.SkipTargetVerification {
```

Comment:
This is saying that TargetVerifier only runs if InlineVerifier runs?

Reply:
good catch - will fix